processes/ingest/doab.py

```diff
@@ -82,3 +83,15 @@ def manage_links(self, record):
         for epubLink in linkManager.ePubLinks:
             ePubPath, ePubURI = epubLink
             self.rabbitmq_manager.sendMessageToQueue(self.file_queue, self.file_route, get_file_message(ePubURI, ePubPath))
+
+    def _process_record(self, record):
```

Comment:
recommend adding a type for the param here and perhaps call it record_mapping so it's not record.record